core/src/tasks/appRefreshQuery/check.ts

```diff
@@ -7,28 +7,28 @@ export class AppRefreshQueryCheck extends CLSTask {
   constructor() {
     super();
     this.name = "appRefreshQuery:check";
-    this.description = "check all appRefreshQueries and run them if it is time";
-    this.frequency =
-      process.env.GROUPAROO_RUN_MODE === "cli:run" ? 0 : 1000 * 60 * 5; // Run every 5 minutes
+    this.description =
+      "run a single appRefreshQuery and update values/trigger schedules if needed";
+    this.frequency = 0;
     this.queue = "apps";
-    this.inputs = {};
+    this.inputs = { appRefreshQuery: { required: true } };
```

Comment:
```suggestion
this.inputs = { appRefreshQueryId: { required: true } };
```

I think this implies that the task expects an actual instance of `appRefreshQuery` to be passed as an Input to the task.  That doesn't really work - task inputs should only be IDs or strings (just like actions).  This /might/ work in test environments, but it won't work for real.  All that gets stored in redis for the task is JSON, so it won't be able to run any of the methods on it. 

The first thing a task should do is load up the appRefreshQuery from the provided appRefreshQueryId, and check if it exists and is ready.  Then, the logic below can continue.